docs/scanner-pools.md

```diff
@@ -0,0 +1,158 @@
+
+- Select the disable action on the node you want to disable
+
+![Disable Scan Node](DisableScanNode.png)
+## Adjusting stake allocation
+
+While it's always possible to add more stake to a pool, it is allocated proportional to the amount of scan nodes in the pool. Total allocated stake can only be `node_count × max_stake_per_scan_node`.
+
+For example, if there are two scan nodes in a pool, a total of 25k FORT deposited and only 5k FORT worth of stake is allowed per scan node, then the pool can allocate a maximum of `5k × 2 = 10k` FORT in total. This is the amount that is effective in calculating the rewards and the rest of `25k - 10k = 15k` remains unallocated until more scan nodes are registered to the pool.
+
+It is up to the pool owner to allocate more from own stake or more from the delegators so the share of the rewards can be adjusted accordingly.
+
+### Adjusting stake
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to adjust the stake allocation
+
+![My Node Pools List](MyNodePoolsList.png)
+
+- Click on "Allocate Stake"
+
+![Allocate Stake On Node](AllocateStakeOnNode.png)
+
+- From here you can either allocate or unallocate stake. To adjust you own stake make sure the subject is *Owner*
+
+![Allocate Modal](AllocationModal.png)
+
+- To adjust the stake of delegators select the *Delegators* subject
+
+![AllocateDelegates](AllocateDelegates.png)
+## Changing the commission
+
+The pool owners can adjust the ratio of the shares they would like to keep for themselves or the delegators, by changing the allocated amounts as described in the previous sections. It can be the case that it's more desirable for the operator to not stake more on the pool but let the delegators fill up the available allocation capacity.
+
+In this case, the commission allows the operators to earn a portion of the delegators' rewards.
+
+!!! warning "Changing Commission"
+    In order to change the commission on a node pool. The pool must be eligible for rewards
+
+- Click on the node pool you would like to adjust the stake allocation (you must be the owner)
+
+- Click on "Change commission rate"
+
+![Change commission Rate](ChangeCommision.png)
+
+- Enter the commission that you would like to set on your pool
+
+![Update commision](UpdateCommissionModal.png)
+
+
+## Claiming rewards
+
+!!! warning "Phase II feature"
+    This is a feature that will be activated and useful in the second phase of the transition to delegated staking. At the time of reading these docs, the feature may not be available on Forta App just yet. The rewards accrued in FORT will be forwarded to pool owner wallets until this feature is available.
+
+After the rewards are calculated at the end of each epoch, they are transferred to the rewards distributor contract. This contract allows the pool owner and the delegators to claim their rewards.
```

Comment:
should we mention that they will have to be claimed via polygonscan? should be provide a polygonscan link directly to the appropriate method? maybe an example would help

Reply:
I thought we were supposed to keep on forwarding the rewards until we start writing the rewards to the contract? If yes, then I think we will have time to add more content to here later.

Reply:
we will be writing rewards to the contract for scanner pools, so we should describe how to claim them

Reply:
OK, got it - we are `reward()`ing only pool owner subjects. I'll update the docs with steps to claim over Polygonscan.